cpp/util/bignum.h

```diff
@@ -0,0 +1,162 @@
+#ifndef CERT_TRANS_UTIL_BIGNUM_H_
+#define CERT_TRANS_UTIL_BIGNUM_H_
+
+#include <glog/logging.h>
+#include <openssl/bn.h>
+
+namespace cert_trans {
+
+
+class BigNum {
+ public:
+  BigNum();
+  explicit BigNum(int64_t w);
+  BigNum(const BigNum& other);
+  ~BigNum();
+
+  BIGNUM* bn();
```

Comment:
This was an escape hatch to still be able to use any `libcrypto` routines (`BN_` or otherwise) which takes a `BIGNUM*`). 

To make your day(s), I've added MOAR bn()!11oneeleventy